tests/inference/ConstructorCall.java

```diff
@@ -3,9 +3,11 @@
 class C {
 	public C(Object o){}
 }
+
 public class ConstructorCall {
-	Object op;
-	//:: fixable-error: (assignment.type.incompatible)
-	//:: fixable-error: (argument.type.incompatible)
-	C c = new @Rep C(op);
+
+	void foo(@Peer Object op) {
```

Comment:
In a separate PR, please add Spottless, like here: https://github.com/eisopux/checker-framework-languageserver/blob/master/build.gradle#L35
and reformat all sources for consistency.

Reply:
Noted